app/views/emancipations/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+  <div class="col-sm-12 dashboard-table-header">
+    <h3>
+      Emancipation Checklist
+    </h3>
+    <noscript>
+      <h3 class="noscript">
+        Javascript is disabled
+      </h3>
+      <p class="noscript">
+        This page requires javascript to work
+      </p>
+    </noscript>
+  </div>
+</div>
+
+<div class="card card-container">
+  <div class="card-body">
+    <% @emancipation_form_data.each do |category| %>
+      <% if category[:category].mutually_exclusive %>
+        <h6>
+          <label for="C<%= category[:category].id %>">
+            <strong> <%= category[:category].name %> </strong>
+          </label>
+          <select id="C<%= category[:category].id %>" class="emancipation-select">
+            <option value="">empty</option>
+            <% category[:options].each do |option| %>
+              <option value="<%= option.id %>" <%= emancipation_select_option_selected(@current_case, option.id) %>>
+                <%= option.name %>
+              </option>
+            <% end %>
+          </select>
+        <h6>
+      <% else %>
+        <h6><strong> <%= category[:category].name %> </strong></h6>
+        <% category[:options].each do |option| %>
+          <input
+            type="checkbox"
+            id="O<%= option.id %>"
+            class="emancipation-check-box"
+            value="<%= option.id %>"
+            <%= emancipation_checkbox_option_checked(@current_case, option.id) %>>
+          <label for="O<%= option.id %>"><%= option.name %></label>
+          <br>
+        <% end %>
+      <% end %>
+    <% end %>
+  </div>
+</div>
```

Comment:
Please don't use the `[:category]` notation, use dot notation instead:

```
category.category.name
```

Reply:
I'll do a grep for `[:` to see if there's more

Reply:
wait that's actually a hash that I made with mapping because I thought it'd be easier to loop. I refactored it so it's just using the relation object

Reply:
oh! sorry. I thought it was an object, since I saw you were using `self[:id]` in other places. No problem if it is a hash.